src/app.js

```diff
@@ -342,12 +349,228 @@ function buildImageImporter() {
 
   $importButton.click(function(){
     activateToolItem($importButton);
-    paper.initImageImport();
+    paper.initImageImport({mode:'trace'});
   });
 
   $('#tools #tool-select').before($importButton);
 }
 
+function buildImageVectorizer() {
+  var $importButton = $('<div>')
+      .addClass('tool')
+      .attr('id', 'importVectorized')
+      .data('cursor-key', 'select')
+      .attr('title', i18n.t('tracing.menutitle'));
+
+  var $imageDiv =  document.createElement("DIV");
```

Comment:
Prefixing variables with `$` is parlance for storing a jQuery variable, though this one is _not_. As you can see on line 359 above, `$('<div>')` is exactly equivalent to this line, creating an unattached DOM div. Any relevant markup that needs extra data can be made this way, otherwise it should likely be added to the HTML directly, and have styles attached. We really shouldn't be setting background images here.